modules/logging-release-notes-5-8-0.adoc

```diff
@@ -0,0 +1,64 @@
+
+* With this update, you can configure the log collector to look for HTTP connections and receive logs as an HTTP server, also known as a webhook. (link:https://issues.redhat.com/browse/LOG-4562[LOG-4562])
+
+* With this update, you can configure audit polices to control which Kubernetes and OpenShift API server events are forwarded by the log collector. (link:https://issues.redhat.com/browse/LOG-3982[LOG-3982])
+
+[id="logging-release-notes-5-8-0-log-storage"]
+=== Log Storage
+* With this update, LokiStack administrator can have more fine grained control over who accesses what logs by granting access to logs on a namespace basis. (link:https://issues.redhat.com/browse/LOG-3841[LOG-3841])
+
+* With this update, the Loki Operator introduces `PodDisruptionBudget` configuration on LokiStack deployments to ensure normal operations during {product-title} cluster restarts by keeping ingestion and query path available. (link:https://issues.redhat.com/browse/LOG-3839[LOG-3839])
+
+* With this update, the reliability of existing LokiStack installations are seamlessly improved by applying a set of default Affinity/Anti-Affinity policies.
+(link:https://issues.redhat.com/browse/LOG-3840[LOG-3840])
+
+* With this update, you can manage zone-aware data replication as an administrator in LokiStack, in order to enhance reliability in the event of a zone failure. (link:https://issues.redhat.com/browse/LOG-3266[LOG-3266])
+
+* With this update, a new supported small-scale LokiStack size of 1x.extra-small is introduced for {product-title} clusters hosting a few workloads and smaller ingestion volumes (upto 100GB/day). (link:https://issues.redhat.com/browse/LOG-4329[LOG-4329])
+
+* With this update, the LokiStack administrator has access to an official Loki dashboard to inspect storage performance and health of each component. (link:https://issues.redhat.com/browse/LOG-4327[LOG-4327])
+
+[id="logging-release-notes-5-8-0-log-console"]
+=== Log Console
+* With this update, you can enable the Logging Console Plugin when Elasticsearch is the default Log Store. (link:https://issues.redhat.com/browse/LOG-3856[LOG-3856])
+
+* With this update, OpenShift Application owners can receive notifications for application logs-based alerts on the OpenShift Developer Console view. (link:https://issues.redhat.com/browse/LOG-3548[LOG-3548])
+
+[id="logging-release-notes-5-8-0-known-issues"]
+== Known Issues
+* Currently, there is a flaw in handling multiplexed streams in the HTTP/2 protocol where you can repeatedly make a request for a new multiplex stream and immediately send an `RST_STREAM` frame to cancel it. This created extra work for the server setting up and tearing down the streams, resulting in a denial of service due to server resource consumption. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/LOG-4609[LOG-4609])
```

Comment:
This CVE seems to have been verified and closed. Any idea why it is under 'known' issues?

Reply:
@kabirbhartiRH Please let me know if you want me to remove this CVE from RNs.

Reply:
@kabirbhartiRH - Docs does not decide which bugs go into the release notes, engineering does. Docs does not track the resolution status of bugs engineering has indicated go into the release notes, so if they close before release notes go out, we would need to be notified by engineering to remove them, Docs would not remove a bug from the release notes on our own.